cmd/podman/play/kube.go

```diff
@@ -144,12 +147,54 @@ func kube(cmd *cobra.Command, args []string) error {
 		}
 		kubeOptions.StaticMACs = append(kubeOptions.StaticMACs, m)
 	}
+	if kubeOptions.Down {
+		return teardown(yamlfile)
+	}
+	return playkube(yamlfile)
+}
 
-	report, err := registry.ContainerEngine().PlayKube(registry.GetContext(), yamlfile, kubeOptions.PlayKubeOptions)
+func teardown(yamlfile string) error {
+	var (
+		podStopErrors utils.OutputErrors
+		podRmErrors   utils.OutputErrors
+	)
+	reports, err := registry.ContainerEngine().PlayKubeDown(registry.GetContext(), yamlfile, kubeOptions.PlayKubeOptions)
 	if err != nil {
 		return err
 	}
 
+	// Output stopped pods
+	fmt.Println("Pods stopped:")
+	for _, stopped := range reports.StopReport {
+		if len(stopped.Errs) == 0 {
+			fmt.Println(stopped.Id)
+		} else {
+			podStopErrors = append(podStopErrors, stopped.Errs...)
+		}
+	}
+	// Dump any stop errors
+	lastStopError := podStopErrors.PrintErrors()
+	if lastStopError != nil {
+		fmt.Fprintf(os.Stderr, "Error: %s\n", lastStopError)
```

Comment:
Do we want to continue if a pod fails to stop? Say if all of the pods in the kube fail to stop, would this exit with code 0?

Reply:
we do want to continue and no, it would exit non-zero